Objects/genobject.c

```diff
@@ -559,6 +561,14 @@ gen_throw(PyGenObject *gen, PyObject *const *args, Py_ssize_t nargs)
     if (!_PyArg_CheckPositional("throw", nargs, 1, 3)) {
         return NULL;
     }
+    if (nargs > 1) {
+        if (PyErr_WarnEx(PyExc_DeprecationWarning,
+                            "the (type, val, tb) exception representation is deprecated, "
+                            "and may be removed in a future version of Python.",
```

Comment:
@gvanrossum "may be" or "will be"?

Reply:
Based on the experience with PEP 585 let’s keep may”.